### PR TITLE
Rename.ztpfw

### PR DESCRIPTION
**Pull request by mareklibra**
# Description

Changes project name from ZTPFW to `Edge cluster` in the UI.

URL of the deployed application is newly: `edge-cluster-setup.apps.[CLUSTER_DOMAIN]`

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Review by iranzo — APPROVED** [review]
LGTM

**Review by jparrill — CHANGES_REQUESTED** [review]
Can we add something in the end of the deploy.sh to verify if the UI was deployed properly and shows fine in the browser? A curl looking for a tag or something similar?

This is just to avoid the customer to face an issue not detected in the CI.

File: deploy-ui/deploy.sh [hunk 1]
```diff
@@ -106,6 +106,12 @@ function deploy_ui() {
     render_file manifests/route.yaml
 }
 
+function verify_ui() {
+    echo ">>>> Verifying deployment of the user interface on: ${spoke}"
+    echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    curl https://${UI_APP_URL} -k -m 30
```
Comment (jparrill) [inline deploy-ui/deploy.sh:112]: UI_APP_URL Already contains https (Line 90).

**Review by jparrill — CHANGES_REQUESTED** [review]
Did you test the script in the environment?

File: deploy-ui/deploy.sh (hunk 1 above)
Comment (mareklibra) [reply deploy-ui/deploy.sh]: fixed, thanks

**Review by jparrill — APPROVED** [review]
If the script was tested, LGTM

**Review by iranzo — APPROVED** [review]
lgtm

**Pull request merged**